centreon/src/Core/HostTemplate/Domain/Model/NewHostTemplate.php

```diff
@@ -187,6 +180,26 @@ public function __construct(
         Assertion::min($highFlapThreshold ?? 0, 0, "{$shortName}::highFlapThreshold");
     }
 
+    /**
+     * Format a string as per domain rules for a host template name.
+     *
+     * @param string $name
+     */
+    public static function formatName(string $name): string
```

Comment:
Avoiding override in children ?
```suggestion
    final public static function formatName(string $name): string
```